src/sentry/seer/anomaly_detection/get_historical_anomalies.py

```diff
@@ -1,24 +1,164 @@
+import logging
+from datetime import datetime
+
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from urllib3.exceptions import MaxRetryError, TimeoutError
+
+from sentry.conf.server import SEER_ANOMALY_DETECTION_ENDPOINT_URL
 from sentry.incidents.models.alert_rule import AlertRule, AlertRuleStatus
 from sentry.models.project import Project
-from sentry.seer.anomaly_detection.types import AnomalyType
+from sentry.net.http import connection_from_url
+from sentry.search.events.types import SnubaParams
+from sentry.seer.anomaly_detection.types import AnomalyDetectionConfig
+from sentry.seer.anomaly_detection.utils import format_historical_data, translate_direction
+from sentry.seer.signed_seer_api import make_signed_seer_api_request
+from sentry.snuba.models import SnubaQuery
+from sentry.snuba.referrer import Referrer
+from sentry.snuba.utils import get_dataset
+from sentry.utils import json
+from sentry.utils.json import JSONDecodeError
+from sentry.utils.snuba import SnubaTSResult
+
+logger = logging.getLogger(__name__)
+
+seer_anomaly_detection_connection_pool = connection_from_url(
+    settings.SEER_ANOMALY_DETECTION_URL,
+    timeout=settings.SEER_ANOMALY_DETECTION_TIMEOUT,
+)
 
 
 def get_historical_anomaly_data_from_seer(
     alert_rule: AlertRule, project: Project, start_string: str, end_string: str
 ) -> list | None:
     """
-    Send time series data to Seer and return anomaly detection response (PLACEHOLDER).
+    Send time series data to Seer and return anomaly detection response.
     """
     if alert_rule.status == AlertRuleStatus.NOT_ENOUGH_DATA.value:
         return []
+    # don't think this can happen but mypy is yelling
+    if not alert_rule.snuba_query:
+        raise ValidationError("No snuba query associated with alert rule.")
+    subscription = alert_rule.snuba_query.subscriptions.first()
+    # same deal as above
+    if not subscription:
+        raise ValidationError("No subscription associated with alert rule.")
+    snuba_query = SnubaQuery.objects.get(id=alert_rule.snuba_query_id)
+    dataset = get_dataset(snuba_query.dataset)
+    window_min = int(snuba_query.time_window / 60)
+    start = datetime.fromisoformat(start_string)
+    end = datetime.fromisoformat(end_string)
+    historical_data = fetch_historical_data(
+        alert_rule=alert_rule, snuba_query=snuba_query, project=project, start=start, end=end
+    )
 
-    return [
-        {
-            "timestamp": 0.1,
-            "value": 100.0,
-            "anomaly": {
-                "anomaly_type": AnomalyType.HIGH_CONFIDENCE.value,
-                "anomaly_value": 100,
+    if not historical_data:
+        raise ValidationError("No historical data available.")
```

Comment:
do we want to raise here?
If there's no historical data, we should jsut return `None` right? (bubbles up to a 400?)

OR - does this mean that there was an issue in the fetch to seer, and the message should actually be some error occured while fetching?